asreview/webapp/src/App.js

```diff
@@ -237,6 +238,21 @@ const App = (props) => {
           <CssBaseline />
 
           <div aria-label="nav and main content">
+            {typeof loginInfo === "string" && loginInfo.length > 0 && (
+              <Alert
+                severity="info"
+                variant="standard"
```

Comment:
Needed?

Reply:
Severity determines the color. Despite te fact that "standard" is the default value for variant, I had to explicitly set it to force the less-obtrusive transparent blue color. The opaque blue seemed standard.